pkgs/by-name/ca/catppuccinifier-cli/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "catppuccinifier-cli";
+  version = "8.0.0";
+
+  src = fetchFromGitHub {
+    owner = "lighttigerXIV";
+    repo = "catppuccinifier";
+    rev = version;
+    hash = "sha256-CEjdCr7QgyQw+1VmeEyt95R0HKE0lAKZHrwahaxgJoU=";
+  };
+
+  sourceRoot = "${src.name}/src/catppuccinifier-cli";
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
```

Comment:
Can't we use the `Cargo.lock` from upstream?

Reply:
Isn't it upstream lock? It's just we can't use git crate with that

Reply:
Fair enough.